spec/garage_spec.rb

```diff
@@ -1,12 +1,27 @@
-require_relative '../lib/garage'
+require_relative "../lib/garage"
+require_relative "../lib/bike_container"
 
 describe Garage do
 
-let(:bike) {Bike.new}
-let(:garage) {Garage.new}
+  let(:garage) {Garage.new}
+  let(:bike) {Bike.new}
+  let(:van) {Van.new}
 
-	it "should fix bikes instantaneously once received" do
-	garage.dock(bike)
-	garage.fix_bikes		
-	end 
-end 
\ No newline at end of file
+  it "should accept a bike and fix it" do
+    #expect garage to be empty
+    expect(garage.bike_count).to eq (0)
+    #lets dock a bike in
+    garage.accept(bike)
+    #we now expect garage to have 1 bike
+    expect(garage.bike_count).to eq (1)
+  end
+
+  it "should release a bike to the van after its fixed" do
```

Comment:
The accept method in bike.rb automatically fix the bike once docked in the garage